src/main/kotlin/net/irisshaders/lilybot/extensions/util/PublicUtilities.kt

```diff
@@ -93,6 +102,7 @@ class PublicUtilities : Extension() {
 
 					val requester = user.asUser()
 					val requesterAsMember = requester.asMember(guild!!.id)
+					// Declare the embed outside the action to allow us to reference it inside the action
```

Comment:
is this not obvious?

Reply:
Not to your average person no. You *must* remember those who read code don't necessarily understand code